.github/workflows/build.yml

```
@@ -34,6 +34,12 @@ defaults:
 env:
   BUILD_ID: ${{ inputs.buildId }}
   BASHBREW_ARCH: ${{ inputs.bashbrewArch }}
+  BASHBREW_IMAGE_SIGNER: sha256:c7df9727f33c9b4b63ffe3214a2628860fc160f9d1ea1d601445058a053088d9
```

As a matter of organization, I think I'd like to see this be the full image reference, especially so we can keep the tag (which is a useful note for us to figure out wtf this value is and how to update it in the future):

```suggestion
  BASHBREW_IMAGE_SIGNER: 'docker/image-signer-verifier:0.5@sha256:c7df9727f33c9b4b63ffe3214a2628860fc160f9d1ea1d601445058a053088d9'
```

and maybe since the image metadata doesn't specify the version, perhaps we go even further all the way to the specific patch release?

```suggestion
  BASHBREW_IMAGE_SIGNER: 'docker/image-signer-verifier:0.5.14@sha256:c7df9727f33c9b4b63ffe3214a2628860fc160f9d1ea1d601445058a053088d9'
```